docs/Config.md

```diff
@@ -1,35 +1,47 @@
 # App configuration
 
 The platform code uses [node-convict](https://github.com/mozilla/node-convict)
-for configuration management. By default, the platform will automatically
-provide a base set of dev-friendly defaults for all non-secret values.
+for configuration management. The platform will automatically provide a base set
+of dev-friendly defaults for all non-secret values.
 
-All config values can be set by a local config file and/or environment vars.
-Local config is preferred in order to not interfere with other apps running in
-the same environment.
+All default values can be overwritten via a local config file or environment
+vars. **Using a config file is preferred** in order to not interfere with other
+apps running in the same environment.
 
-An example local config file can be found in
-[`config.test.json`](../config.test.json) - this is the file used to set env
-values for unit tests. The 'secret' values are dummy values that will not work
-for actual development, so you will need to provide the real values in a config
-file.
+## Configuration precedence
 
-1. The OAuth consumer `key` for the platform
-2. The OAuth consumer `secret`
-3. The photo scaler salt for Meetup Classic
+Configuration values are read in the following order of increasing precedence:
+
+1. Default value - see `src/util/config/index.js` and `src/util/config/build.js`
+2. Application root `config.<NODE_ENV>.json` file
+3. Environment variable in `$HOME/.mupweb.config`
```

Comment:
Technically it doesn't matter where the env var comes from - not sure if it would be confusing to people if `.mupweb.config` wasn't automatically read, or conversely if they had set a relevant env var in another way that wasn't in `.mupweb.config`. Not a big deal.